Client/Application/AppDelegate.swift

```diff
@@ -851,30 +853,29 @@ class AppSyncDelegate: SyncDelegate {
 
         // check to see what the current notification settings are and only try and send a notification if
         // the user has agreed to them
-        if let currentSettings = app.currentUserNotificationSettings {
-            if currentSettings.types.rawValue & UIUserNotificationType.alert.rawValue != 0 {
+        UNUserNotificationCenter.current.getNotificationSetings { settings in
+            if settings.alertSetting == UNNotificationSetting.enabled {
                 if Logger.logPII {
                     log.info("Displaying notification for URL \(url.absoluteString)")
                 }
 
-                let notification = UILocalNotification()
-                notification.fireDate = Date()
-                notification.timeZone = NSTimeZone.default
+                let notificationContent = UNMutableNotificationContent()
                 let title: String
                 if let deviceName = deviceName {
                     title = String(format: Strings.SentTab_TabArrivingNotification_WithDevice_title, deviceName)
                 } else {
                     title = Strings.SentTab_TabArrivingNotification_NoDevice_title
                 }
-                notification.alertTitle = title
-                notification.alertBody = url.absoluteDisplayString
-                notification.userInfo = [TabSendURLKey: url.absoluteString, TabSendTitleKey: title]
-                notification.alertAction = nil
+                notificationContent.title = title
+                notificationContent.body = url.absoluteDisplayString
+                notificationContent.userInfo = [TabSendURLKey: url.absoluteString, TabSendTitleKey: title]
+                notificationContent.categoryIdentifier = TabSendCategory
+
+                let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 1, repeats: false)
```

Comment:
Is this 1 second?

1. please use `.seconds(1)`
2. can we post the notification immediately? If the answer is no, please comment why.